tests/Controller/TaxonApiTest.php

```diff
@@ -431,4 +432,78 @@ public function it_allows_paginating_the_index_of_taxons()
         $response = $this->client->getResponse();
         $this->assertResponse($response, 'taxon/paginated_index_response');
     }
+
+    /**
+     * @test
+     */
+    public function it_allows_to_update_position_of_product_in_a_taxon()
+    {
+        $this->loadFixturesFromFile('authentication/api_administrator.yml');
+        $this->loadFixturesFromFile('resources/products.yml');
+        $productTaxons = $this->loadFixturesFromFile('resources/product_taxons.yml');
+
+        /** @var TaxonInterface $taxon */
+        $taxon = $productTaxons['mugs'];
+
+        $data =
+<<<EOT
+        {
+            "positionsData": [
+                {
+                    "product_code": "MUG_SW",
+                    "position": 2
+                },
+                {
+                    "product_code": "MUG_BB",
+                    "position": 0
+                }
+            ]
+        }
+EOT;
+
+        $this->client->request('PUT', sprintf('/api/v1/taxons/%s/products', $taxon->getCode()) , [], [], static::$authorizedHeaderWithContentType, $data);
```

Comment:
It could also be extracted to the private method... But it is just a small DX improvement, to have only one place to change a path in case we would change something